acceptance-tests/src/test/java/tech/pegasys/ethsigner/tests/dsl/signer/EthSignerProcessRunner.java

```diff
@@ -139,6 +140,17 @@ public void start(final String processName) {
       params.add("--data-path");
       params.add(dataPath.toAbsolutePath().toString());
     }
+    if (signerConfig.serverTlsOptions().isPresent()) {
+      final TlsOptions serverTlsOptions = signerConfig.serverTlsOptions().get();
+      params.add("--tls-keystore-file");
+      params.add(serverTlsOptions.getKeyStoreFile().toString());
+      params.add("--tls-keystore-password-file");
+      params.add(serverTlsOptions.getKeyStorePasswordFile().toString());
+      if (serverTlsOptions.getKnownClientsFile().isPresent()) {
+        params.add("--tls-client-whitelist-file");
```

Comment:
didn't you end up updating it to "known_clients"??

Reply:
ah yes .. oops.